modules/etc/smartics/camel-modules.xml

```diff
@@ -433,6 +437,38 @@
             <module name="javax.api"/>
             <module name="org.osgi.core"/>
             <module name="org.slf4j"/>
+            <module name="rhino" export="true">
```

Comment:
Updated to include script languages: rhino:js, python, jruby, groovy to camel-script:
https://github.com/wildfly-extras/wildfly-camel/pull/489

> Why does the mqtt component need to reexport paths from rhino?
It's not mqtt but camel-script, same reason why bsh is already exported.